scripts/containers.sh

```diff
@@ -12,6 +12,7 @@ start() {
     ./scripts/devicesimulation.sh start
     ./scripts/telemetry.sh start
     ./scripts/iothubmanager.sh start
+	./scripts/asamanager.sh start
```

Comment:
indentation - you might want to check you text editor, tabs growing everywhere 🌵